<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcul adresse IP</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>294</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Adresse IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIP3">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIP2">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIP1">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxIP0">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Masque CIDR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxMasque">
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Calcul des paramétres réseau </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCalcul">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcul</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_Masque">
      <property name="text">
       <string>Masque</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Reseau">
      <property name="text">
       <string>Réseau</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Diffusion">
      <property name="text">
       <string>Diffusion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Nbre">
      <property name="text">
       <string>Nombre ip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_IPmini">
      <property name="text">
       <string>Adresse ip mini</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_IPmaxi">
      <property name="text">
       <string>Adresse ip maxi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string> d'une adresse IP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
